frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -459,7 +459,6 @@ machinery-MicrosoftTerminology--visit-microsoft = MICROSOFT
         Visit Microsoft Terminology Service API.
         © 2018 Microsoft Corporation. All rights reserved.
 
-
```

Comment:
Please revert this empty line you removed. :)

Reply:
Oops!